.circleci/config.yml

```diff
@@ -8,7 +8,7 @@ workflows:
 jobs:
   run-tests:
     docker:
-      - image: aotuai/brainframe_build_env:0.28.1
+      - image: aotuai/brainframe_build_env:0.28.0-openvino-2021.3-igpu
```

Comment:
Is this a temporary tag?

Reply:
Yeah. This PR needs https://github.com/aotuai/brainframe/pull/1492 to be merged in before we can fix it up. 

I have the checklist for the PR include fixing this tag, and also waiting for the aforementioned PR to be merged.